### PR TITLE
feat(pkh-tezos): remove @taquito/utils for stablelib

### PR DESCRIPTION
**Pull request by zachferland**
need to test/review further

### CODE REVIEW

**Comment by zachferland** [comment]
Clearly not working, add simple tests for the two funcs i tried to implement 

maybe some differences between handling uin8arrays/enc/dec, nothing obviously wrong yet

File: packages/pkh-tezos/test/index.test.ts [hunk 1]
```diff
@@ -1,5 +1,28 @@
-describe('Solana Cacao Auth Verify', () => {
-  test.skip('Verify', () => {
+import { InMemorySigner } from '@taquito/signer'
+import { verifySignature, getPkhfromPk } from '../src/verifier'
+import { char2Bytes, verifySignature as taquitoVerifySignature, getPkhfromPk as taquitoGetPkhfromPk } from '@taquito/utils'
```
Comment (oed) [inline packages/pkh-tezos/test/index.test.ts:3]: Can we just generate some test vectors so that we don't need to include the `@taquito` deps at all?

**Review by oed — CHANGES_REQUESTED** [review]
Lgtm, just one request

**Review by oed — APPROVED** [review]
Lets actually merge this and get into js-ceramic asap. We can address test vectors later

**Pull request merged**